go/test/endtoend/vtgate/queries/aggregation/aggregation_test.go

```diff
@@ -31,6 +31,9 @@ import (
 )
 
 func start(t *testing.T) (utils.MySQLCompare, func()) {
+	// ensure that the vschema and the tables have been created before running any tests
+	_ = utils.WaitForAuthoritative(t, keyspaceName, "t1", clusterInstance.VtgateProcess.ReadVSchema)
+
```

Comment:
Is this something we also want on the older release branches to avoid flaky tests there?

Reply:
whenever we have established that the schema tracking is what is introducing the flakiness, this is an excellent way to stop if from flaking